cvat-core/src/server-proxy.ts

```diff
@@ -373,6 +373,50 @@ function setAuthData(response: AxiosResponse): void {
     }
 }
 
+function fetchAggregatedResource<FilterType, SerializedDataType>(
```

Comment:
1. fetchAggregatedResource does not seem to be a correct name for function fetching all resources (the same about parameter `aggregate`. `aggregation` is another operation, when many values are converted to a certain value)

2. It is not really logical to handle single `id` logic in this function

3. I also do not think we should handle case when aggregate===false in this function

You probably did that to reduce possible code duplication, but I think the code seems more difficult to understand not and it is not worthy of profit.  

Why not call `fetchAll` from corresponding functions when `aggregate` (or another, more suitable name, is `true`? 

e.g. 

if (aggregate) {
    return fetchAll(url, filter);
}

Just three lines of code.

Reply:
Ok, added.